src/docs/cookbook/maintenance/error-reporting.md

```diff
@@ -47,138 +45,64 @@ To get a DSN, use the following steps:
 
   1. [Create an account with Sentry][].
   2. Log in to the account.
-  3. Create a new app.
-  4. Copy the DSN.
+  3. Create a new Flutter project.
+  4. Copy the code snippet which includes the DSN.
```

Comment:
The installation step has a snippet which includes the DSN for the actual project:

![image](https://user-images.githubusercontent.com/1633368/107383707-4393f580-6abf-11eb-91f2-71f296babf5e.png)